wolfcrypt/benchmark/benchmark.c

```diff
@@ -3596,16 +3967,32 @@ static void bench_aesgcm_internal(int useDeviceID,
                     }
                 }
             } /* for i */
+            RECORD_MULTI_VALUE_STATS();
         } /* for times */
         count += times;
-    } while (bench_stats_check(start));
+    } while (bench_stats_check(start)
+#ifdef MULTI_VALUE_STATISTICS
+           || runs < minimum_runs
+#endif
+           );
+
 exit_aes_gcm:
     bench_stats_sym_finish(encLabel, useDeviceID, count, bench_size,
                            start, ret);
+#ifdef MULTI_VALUE_STATISTICS
+    bench_multi_value_stats(max, min, sum, squareSum, runs);
+#endif
 
 #ifdef HAVE_AES_DECRYPT
     XMEMSET(dec, 0, sizeof(dec));
 
+#ifdef MULTI_VALUE_STATISTICS
+    prev = 0;
```

Comment:
This reset code pattern occurs over and over, and tracks the declarations from `DECLARE_MULTI_VALUE_STATS_VARS()`, so it should also be done with a macro.

Reply:
Done!